source/install/smtp-email-setup.rst

```diff
@@ -182,6 +180,10 @@ Checking your SMTP server is reachable
        250-STARTTLS
        250-PIPELINING
        250 8BITMIME
+       
+
+.. note:: 
+  We're not installing telnet by default on the official docker images, so you either need to use ``ping`` on those or install telnet yourself either directly in the machine or by modifying the dockerfile.
```

Comment:
```suggestion
  As we're not installing telnet by default on the official docker images you either need to use ``ping`` on those or install telnet yourself either directly modifying the dockerfile.
```

Not sure what machine we're referring to so I removed it - happy to put it back if we can clarify.